scanner/src/main/kotlin/scanners/fossid/FossIdUrlProvider.kt

```diff
@@ -144,9 +152,12 @@ internal class FossIdUrlProvider private constructor(
             if (replaced != repoUrl) {
                 logger.info { "URL mapping applied to $repoUrl: Mapped to $replaced." }
                 return replaceVariables(replaced)
+            } else {
+                logger.debug { "Mapping $regex did not match." }
             }
         }
 
+        logger.info { "No matching URL mapping could be found for '$repoUrl'." }
```

Comment:
Nit: Empty line after this one to separate from the unrelated `return`  statement.